tests/vse_sync_pp/test_requirements.py

```diff
@@ -13,18 +13,34 @@ class TestRequirements(TestCase):
     """Test cases for vse_sync_pp.requirements.REQUIREMENTS"""
     def test_g8272_prtc_a(self):
         """Test G.8272/PRTC-A requirement values"""
-        self.assertEqual(
-            REQUIREMENTS['G.8272/PRTC-A'],
-            {
-                'time-error-in-locked-mode/ns': 100,
-            },
-        )
+
+        self.assertEqual(REQUIREMENTS['G.8272/PRTC-A']['time-error-in-locked-mode/ns'], 100)
+
+        (a, f1), (c, f2) = REQUIREMENTS['G.8272/PRTC-A']['maximum-time-interval-error-in-locked-mode/us'].items()
+
+        self.assertEqual(f1(100), 0.052500000000000005)
```

Comment:
`0.525...` is very specific, worth a comment? (For one, I'd like to understand why!)